src/app/pages/send/send-crypto-asset-form/family/stacks/hooks/use-stacks-broadcast-transaction.tsx

```diff
@@ -27,10 +32,10 @@ import { useSignTransactionSoftwareWallet } from '@app/store/transactions/transa
 
 import { convertToMoneyTypeWithDefaultOfZero } from '../../../components/confirmation/send-form-confirmation.utils';
 
-export function useStacksBroadcastTransaction(unsignedTx: string) {
+export function useStacksBroadcastTransaction(unsignedTx: string, token: CryptoCurrencies = 'STX') {
```

Comment:
When it comes to currencies, I'd avoid setting a default. Even when using STX, let's be explicit. 

Consider the case that another developers comes and uses this for a SIP token. They forget to add a `token` (there's no type error because there's a default), and it works fine, with wrong values, because of the default.

Imo, it has the potential to cause more problems, than benefit it brings.

Reply:
yeah, agree here